apps/finance/contracts/Finance.sol

```diff
@@ -134,7 +134,7 @@ contract Finance is AragonApp, ERC677Receiver {
 
     /**
     * @dev Deposit for ERC20 approved tokens
-    * @notice Send `_amount` `_token`
+    * @notice Send `_amount` `_token.symbol()`
```

Comment:
Needs to be suffixed with `: string`, same goes for every other place you do `_token.symbol()`. This is because Radspec has no ABIs at all, so it needs to be inlined.